plugins/woocommerce/changelog/update-pricing-inputs-width

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: enhancement
+
+Adapt the width of pricing fields in new product management experience
```

Comment:
I'd probably make this comment a little more specific. "Adapt" feels a bit vague.